City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-3899: Add js functionality that disables other selects if one is …

…selected

How to test:

1. Checkout this branch and also require this branch: `composer require drupal/hdbt:dev-UHF-3899_high_school_search_filter_changes`.
2. `make drush-cr`
3. `make shell`
4. `drush locale-import fi /app/public/themes/custom/hdbt_subtheme/translations/fi.po`
5. `drush locale-import sv /app/public/themes/custom/hdbt_subtheme/translations/sv.po`
6. Import keywords from TPR using this command: `drush migrate:import tpr_ontology_word_details`
7. `drush cr`
8. For reason unknown to man keyword import needs to be done twice on the first time: `drush migrate:import tpr_ontology_word_details`
9. `drush cr`
10. Log into the site and go to `/admin/config/school-settings` path and set the year to 2021.
11. Create high school search and make sure there is enough items so that you get some filters for the paragraph.
12. If you select one filter the other one should turn disabled. If you return the value to "All" then the filter that was disabled should be usable again.
13. The disabled select should look like it does in hds (https://hds.hel.fi/components/dropdown).
14. Test the functionality with screen reader as well. You should get a aria-live information after you select a value on the filter.

FYI: Still missing Swedish translations for the help-text so won't be merging before we have those. 

Also check this one:
https://github.com/City-of-Helsinki/drupal-hdbt/pull/227
